schema/workflow.json

```diff
@@ -442,17 +442,14 @@
           "description": "Reference to the unique name of a 'consumed' event definition"
         },
         "data": {
-          "type": [
-            "string",
-            "object"
-          ],
-          "description": "If string type, an expression which selects parts of the states data output to become the data (payload) of the event referenced by 'triggerEventRef'. If object type, a custom object to become the data (payload) of the event referenced by 'triggerEventRef'."
+          "$ref": "common.json#/definitions/objectOrMarkup",
+          "description": "Object structure that can use expressions to generate the data of the produced event."
         },
         "contextAttributes": {
           "type": "object",
-          "description": "Add additional extension context attributes to the produced event",
+          "description": "An object carrying additional extension context attributes to the produced event that may each use expresions",
```

Comment:
again, i really do not understand this sentence

Reply:
tried to clarify above. maybe there's a less technical description that is easier to understand. any suggestions?